components/src/core/components/SchemaForm/SchemaForm.vue

```diff
@@ -177,9 +189,12 @@ export default defineComponent({
         // https://github.com/vuejs/core/blob/9c304bfe7942a20264235865b4bb5f6e53fdee0d/packages/runtime-core/src/h.ts#L137
         // maybe able to refactor with vue upgrade
         return h(
-          component as DefineComponent,
+          // eslint-disable-next-line @typescript-eslint/no-explicit-any
+          component as DefineComponent<any>,
```

Comment:
Isn't TS allows us to define generic types?

Reply:
Typescript have generics, but here the types are for props of this component. if we pass a generic type i don't know how to type it in the schema. if we don't use any, it works now.

I referenced the type here
https://github.com/vuejs/core/blob/9c304bfe7942a20264235865b4bb5f6e53fdee0d/packages/runtime-core/src/h.ts#L159